.github/workflows/workflow.yml

```diff
@@ -129,6 +142,13 @@ jobs:
     steps:
     - uses: Brightspace/third-party-actions@actions/checkout
 
+    - if: ${{ inputs.artifacts_url != '' }}
+      name: 'Extract artifacts'
+      uses: Brightspace/s3-artifact-actions/extract@master
+      with:
+        path: .artifacts
+        artifacts_url: ${{ inputs.artifacts_url }}
```

Comment:
Don't think you need this, since it will get collected as part of the plan?

https://github.com/Brightspace/terraform-workflows/blob/main/actions/plan/plan.sh#L104-L109